spieleplattform-backend/src/main/kotlin/com/example/spieleplattformbackend/models/Rating.kt

```diff
@@ -6,9 +6,13 @@ import jakarta.persistence.*
 @Table(name = "ratings")
 class Rating(
     @Column(nullable = false)
-    var ratingRange: Int,
+    var ratingValue: Int,
+    @Column(nullable = true)
     var comment: String,
     //var game: Game,
```

Comment:
Hier ist noch ein unnötiger Kommentar. Auskommentierten Code würde man auch generell nicht einchecken/nach develop mergen. Im Übungsprojekt ist das jetzt nicht so tragisch, aber in einem echten Projekt, sollte auskommentierter Code eigentlich nie eingecheckt werden. Auch wenn man sich nicht sicher ist, ob man zuvor gelöschten Code nicht vielleicht doch noch einmal braucht, ließe sich dieser ja über die git Historie wiederherstellen.